kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -6,12 +6,30 @@ if [ $# -ne 3 ]; then
     exit 1
 fi
 
+cluster_name=`kubectl config get-clusters | grep -v NAME`
+
 ns=$1
 srv=$2
 conf=$3
 secretref=${4:-https://openstack.cern.ch:9311/v1/secrets/35145b52-18af-47a3-90d0-1861c51a9c65}
 
+if [[ "$cluster_name" == cmsweb-test[1-9]* ]] ; then
```

Comment:
I never used regex in `if` statements, so I want to double check that it works here.